LICENSE

```diff
@@ -1,6 +1,6 @@
 Attribution 4.0 International
 
-Copyright (c) 2020-2023, BIDS Contributors.
+Copyright (c) 2020-2024, BIDS Contributors.
```

Comment:
From what @effigies once told me, we can just delete the date part of this.

Can't find the link anymore but it is not something that actually matters.

Reply:
https://hynek.me/til/copyright-years/

Reply:
```suggestion
Copyright (c) 2020, BIDS Contributors.
```